x-pack/solutions/security/plugins/security_solution/public/data_view_picker/redux/reducer.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { combineReducers } from '@reduxjs/toolkit';
+
+import { DataViewPickerScopeName } from '../constants';
+import {
+  createDataViewSelectionSlice,
+  initialScopeState,
+  initialSharedState,
+  shared,
+} from './slices';
+
+export const scopes = {
+  [DataViewPickerScopeName.default]: createDataViewSelectionSlice(DataViewPickerScopeName.default),
+  [DataViewPickerScopeName.timeline]: createDataViewSelectionSlice(
+    DataViewPickerScopeName.timeline
+  ),
+  [DataViewPickerScopeName.detections]: createDataViewSelectionSlice(
+    DataViewPickerScopeName.detections
+  ),
+  [DataViewPickerScopeName.analyzer]: createDataViewSelectionSlice(
+    DataViewPickerScopeName.analyzer
+  ),
+} as const;
+
+export const dataViewPickerReducer = combineReducers({
```

Comment:
nit: can we call this `securityDataViewManager` or just `dataViewManager` or something similar rather than a reference to the component?

Reply:
done